### DISCUSSION TITLE
Aktuelles install.sh (11/2023)

### DISCUSSION THREAD

**Discussion by zapfs** [Schrolli91/BOSWatch#563] [General]
Hat irgendjemand ein Installskript, welches auf einem neuen RasPi 3 läuft? Momentan bricht das aktuelle bei qt4 und Python-pip ab.

**Comment by stoepf** [comment]
Probier's mal damit https://github.com/Schrolli91/BOSWatch/blob/develop/install_bullseye.sh

**Reply by zapfs** [reply]
Danke für den Tipp. Installation hat geklappt.
